lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dva/DvaThirdPartyDocumentGateway.java

```diff
@@ -259,7 +257,7 @@ private DocumentCheckResult responseHandler(
                 documentCheckResult.setValid(unwrappedDvaResponse.isValidDocument());
 
                 return documentCheckResult;
-            } catch (IpvCryptoException e) {
+            } catch (IpvCryptoException | java.text.ParseException e) {
```

Comment:
Can you add import back in for the ParseException just for readability?

Reply:
The issue here is we have an apache parse exception slightly higher up thats gets imported